### ISSUE TITLE
No error reported using `code ` with non private url

### ISSUE THREAD

**Issue by danielzen** [blinksh/blink#1882]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration
iPadOS 17.0.2
Blink 17.0.1.839 TestFlight

### Describe the bug

I type `code https://<myurl>` and get back absolutely nothing.

In the end I forgot to use a custom port (I am not using 443), but the complete lack of response threw me. In the browser, with the same url I would get:
```
You connection is not private
NET::ERR_CERT_INVALID
```

Some error response indicating that the connection was not made.

**Comment by carloscabanero** [comment]
Good catch, will see what we can do. We are actually thinking about separating Build and Code in its own app, as it will give us more flexibility with what we can do with this.
